### DISCUSSION TITLE
main repo

### DISCUSSION THREAD

**Discussion by fcomte** [InseeFrLab/legacy-onyxia-entrypoint#59] [Polls]
For now, the main repository of Onyxia is this one. It's just a meta project to link to other real ones.
For the visibility of the project we are thinking about the possibilty to promote [onyxia-web](https://github.com/InseeFrLab/onyxia) as the main repo of the Onyxia project.

This project evolved fast and is the main experience for the users.

Do you think `onyxia-web` should be renamed `onyxia` and the meta project should be archived ?

**Comment by gmougeolle** [comment]
On a side note, I find that `onyxia-ui` is a bit confusing, but maybe having the actual UI as the main project will clear that up

**Reply by garronej** [reply]
Yes,  
Thank you for your feedback.  
The confusing part is that, in the helm values.json, the `api` subchart is `InseeFrLab/onyxia-api` and the `ui` subchart is `InseeFrLab/onyxia-web`.  
We will change that.  
`onyxia-web` will become `onyxia` and `ui` will renamed to `front` or `spa`.  
I think it will clarifies things.  

Also, we are going to improve he onyxia helm chart. There will be only one public version of Onyxia unlike today where you have the version of `onyxia-api` and the version of `onyxia-web` and there is no way to know which is compatiblie with with.  

I hope this futures changes will make things less confusing for everyone.
